src/test/setup_common.cpp

```diff
@@ -82,8 +82,8 @@ TestingSetup::TestingSetup(const std::string& chainName) : BasicTestingSetup(cha
 
     mempool.setSanityCheck(1.0);
     pblocktree.reset(new CBlockTreeDB(1 << 20, true));
-    pcoinsdbview.reset(new CCoinsViewDB(1 << 23, true));
-    pcoinsTip.reset(new CCoinsViewCache(pcoinsdbview.get()));
+    g_chainstate.reset(new CChainState(
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Suggest `x = MakeUnique<X>()` instead of `x.reset(new X())` to avoid raw pointers and new in C++, which are suspicious.